looprpc/client.proto

```diff
@@ -80,6 +80,15 @@ service SwapClient {
     */
     rpc GetL402Tokens (TokensRequest) returns (TokensResponse);
 
+    /* loop: `listauth`
```

Comment:
I think we can remove the `loop: listauth` part, as that's for the API docs generator only, which should use the new RPC.

Reply:
Fixed